src/content/docs/tutorial-service-level-mgmt/intro-to-slm.mdx

```diff
@@ -0,0 +1,156 @@
+    id="slm-terminology-service-level"
+    title="Service level"
+  >
+    A [service level](https://en.wikipedia.org/wiki/Service_level) measures the performance of a system. You measure service levels by using indicators and comparing them against a set of objectives for expected performance and outcomes.
+  </Collapser>
+
+  <Collapser
+    id="slm-terminology-service-level-management"
+    title="Service level management"
+  >
+    We define service level management as the practice of reporting, maintaining, and managing service level compliance.
+  </Collapser>
+
+  <Collapser
+    id="slm-terminology-sli"
+    title="Service level indicator"
+  >
+    A service level indicator (SLI) measures a service level. An SLI identifies something you want to measure, and measures either one datapoint or a composite of datapoints. An SLI most commonly represents a percentage (%) of successful transactions or events. It measures the total number of good events over the total number of valid events calculated over a specified period of time, such as 1 hour, 1 day, or 1 week.
+
+    **Examples SLIs:**
+    * Percentage of login API transactions that complete within 500 milliseconds.
+    * Percentage of login API transactions that complete without an internal server error.
+    * Percentage of login API transactions that complete within 500 milliseconds and without an internal server error.
+    * Percentage of synthetic check Pings against the Login API that are successful.
+    * Percentage of browser logins that transition to the landing page within 3 seconds.
+    * Percentage of metrics ingested that are available for query within 3 seconds.
+    * Some or all of the above aggregated into a single SLI.
+  </Collapser>
+
+  <Collapser
+    id="slm-terminology-slo"
+    title="Service level objective"
+  >
+    A service level objective (SLO) describes the expected service level of a system, job, or capability. It usuall indicates the required percentage of success of one or more SLIs within a given period of time.
+
+    **Example SLOs:**
+    * The login API will respond within 500 milliseconds and error free 99.99% of the time each week.
+    * The mobile login capability will transition to the landing page within 3 seconds 99.99% of the time each week.
+    * The data ingest of metrics will be available for query within 3 seconds of consumption from the API 99.99% of the time each day.
+  </Collapser>
+
+  <Collapser
+    id="slm-terminology-service-boundary"
+    title="Service boundary"
+  >
+    A service boundary is the point in a system where the consumer client connects to make requests, also known as the external API or endpoint. Service boundaries can also be internal between two distinct systems, where one collection of applications serves the requests of another collection of applications. For example, an identity management system can serve for both login requests from the client and permissions authorization for different features.
```

Comment:
Hm, is there a more human way to describe this? consumer-client seems a little opaque. 

i think b/c this content is so conceptual, simplifying with concrete examples will more easily convey whats actually happening and how it relates

Reply:
Simplified a bit, thank you